bookwyrm/templates/book/book_identifiers.html

```diff
@@ -4,42 +4,48 @@
 {% if book.isbn_13 or book.oclc_number or book.asin or book.aasin or book.isfdb %}
 <dl>
     {% if book.isbn_13 %}
-        <div class="is-flex">
+        <div class="is-flex is-flex-wrap-wrap">
             <dt class="mr-1">{% trans "ISBN:" %}</dt>
-            <dd itemprop="isbn">{{ book.isbn_13 }}</dd>
+            <dd itemprop="isbn" class="mr-1" id="isbn_content">{{ book.isbn_13 }}</dd>
+            <div>
+                <button class="button is-small" data-copywithtooltip data-content-id="isbn_content" data-tooltip-id="isbn_tooltip">
+                    <span class="icon icon-copy" title="{% trans "Copy ISBN" %}"></span>
```

Comment:
As is, this button is unlabelled for me when I use my screen reader. You can add a span that contains text for the screen reader to the icon's span using the class `is-sr-only` (ie, is screen reader only), something like:
```
<span class="icon icon-copy" title="{% trans "Copy ISBN" %}">
    <span class="is-sr-only">{% trans "Copy ISBN" %}</span>
</span>
```

Reply:
Sure it seems to work with the chromevox extension on my browser, but I added that span you wanted so hopefully that should work on your reader too now.

Reply:
Ah, how fickle the different brands of screen reader are! I'll have to give chromevox a try, I test with Apple's built-in VoiceOver